app/controllers/events_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  before_action :require_user, except: [:index]
+
+  def new
+    @users = User.all
+    @users = current_user.except_current_user(@users).pluck(:name)
+    @event = Event.new
+  end
+
+  def show
+    @event = Event.find_by(id: params[:id])
+  end
+
+  def index
+    @events = Event.all
+  end
+
+  def create
+    @users = User.all
+    @users = current_user.except_current_user(@users).pluck(:name)
+    @event = current_user.events.build(event_params)
+    if @event.save
+      create_attendees
```

Comment:
What happens if an event is created without any attendees? 

![image](https://user-images.githubusercontent.com/22228397/65054024-86858400-d932-11e9-8e09-6e5edcc43d14.png)

User must be able to create an event.

[`REQUIRED`]

Reply:
Event is still committed to the database though:

![image](https://user-images.githubusercontent.com/22228397/65054264-e9771b00-d932-11e9-9638-41ee371751f5.png)

Reply:
This is an error on my part have fixed this